calculator.go

```diff
@@ -1,13 +1,81 @@
 // Package calculator provides a library for simple calculations in Go.
 package calculator
 
+import (
+	"fmt"
+	"math"
+)
+
 // Add takes two numbers and returns the result of adding them together.
 func Add(a, b float64) float64 {
-return a + b
+	return a + b
 }
 
 // Subtract takes two numbers and returns the result of subtracting the second
 // from the first.
 func Subtract(a, b float64) float64 {
-	return b - a
+	return a - b
+
+}
+
+func Multiply(a, b float64) float64 {
```

Comment:
Good! Now we're about ready to write some documentation comments for the remaining functions, aren't we?